lib/tasks/migrate.rake

```diff
@@ -455,4 +455,14 @@ namespace :migrate do
       end
     end
   end
+
+  namespace :projects do
+    desc 'Set default value for authentication invitation' do
```

Comment:
This may not be needed anymore after reading that post on the change in PG V11+ behaviour as for rows that don't have that column value, it appears to use the default value when fetching rows and or updating existing rows.
```
Any row that doesn’t have that column must have existed before the table change was made, and uses this value stored in the catalog when the row is fetched.
...
Likewise, any time that a row is updated the column’s value is filled in on the new row version, so over time the proportion of rows using the value fetched from the catalog will decrease.
```

There is now harm in backfilling the data and tbh it could be a useful example of how to run rake tasks etc but it may not be needed now.

I'll leave it up to you to decide to include it or not (and run it via rake task etc).

Reply:
Thanks for looking into this! I think I'll remove it.